### PR TITLE
Updated package.json

### PR DESCRIPTION
**Pull request by brianvoe**
In order to use your package babel-preset-es2015 is necessary. I would suggest moving to babel-preset-env for future developmet.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
   "author": "Handsoncode",
   "main": "src/HotTable.vue",
   "dependencies": {
-    "handsontable": "^0.33.0"
+    "handsontable": "^0.33.0",
+    "babel-preset-es2015": "^6.24.0"
```
Comment (kimond) [inline package.json]: I think this dependency should be in the devDependencies.

**Comment by AMBudnik** [comment]
Thank you for your time to test it. Can you move the changes to `develop` branch, please?

**Comment by brianvoe** [comment]
Ok marge branch was changed to develop

**Comment by AMBudnik** [comment]
Thanks!

Our Vue developer is currently fixing the mergeCells plugin but after he's back to Vue he'll surely review your PR.

**Comment by mismith** [comment]
Any luck now @AMBudnik? Is he back now? Currently this plugin is unusable with the default webpack vue-cli template...

**Comment by AMBudnik** [comment]
Sorry, he had a smaller gap between two bigger tasks than we previously thought and is now finishing the second one. If you would like to be up to date with our tasks here's the place to visit https://trello.com/b/PztR4hpj/handsontable-roadmap

**Pull request closed without merging**